### DISCUSSION TITLE
[Formatting] Operators at the beginning of lines

### DISCUSSION THREAD

**Discussion by Followin** [rome/tools#4373] [Suggestions]
This should be an option/default behavior: 

Before

```javascript
var a = '123354535435' + '2342423153';
```

After (if it's too long to fit in a line)
```javascript
var a = 
    '123354535435'
    + '2342423153';
```

instead of
```javascript
var a = 
    '123354535435' +
    '2342423153';
```

Same goes for all of the conditional operators like `&&` and `||`

Related prettier discussion that forced people into creating their own forks: https://github.com/prettier/prettier/issues/3806

**Comment by mikez** [comment]
Learned about Rome today. Impressed by the speed and defaults. 🚀

Not having support for breaking before binary operators is a blocker for me.
To [quote](https://peps.python.org/pep-0008/#should-a-line-break-before-or-after-a-binary-operator) Guide van Rossum: "For decades the recommended style was to break after binary operators. But this can hurt readability in two ways [...]".

This new style is the default in Python for example; see [PEP 8](https://peps.python.org/pep-0008/#should-a-line-break-before-or-after-a-binary-operator) and [black formatter](https://github.com/psf/black).

Read more here for both sides of this:

- https://peps.python.org/pep-0008/#should-a-line-break-before-or-after-a-binary-operator
- https://softwareengineering.stackexchange.com/questions/160522/should-i-put-newlines-before-or-after-binary-operators

(/cc @MichaReiser @ematipico)
